nexus/auth/src/authz/omicron.polar

```diff
@@ -255,7 +257,7 @@ has_permission(actor: AuthenticatedActor, _perm: String, silo_user: SiloUser)
     if actor.equals_silo_user(silo_user);
 
 has_permission(actor: AuthenticatedActor, "read", silo_user: SiloUser)
-    if silo_user.silo in actor.silo;
+    if actor.is_user and silo_user.silo in actor.silo;
```

Comment:
hm, won't the scim actor need to read the users? or will it get that permission from some other part of the config?

Reply:
That will be added in the next PR, as that will actually do the user and group modification.